Cesium.CodeGen/Ir/BlockItems/WhileStatement.cs

```diff
@@ -0,0 +1,76 @@
+using Cesium.CodeGen.Contexts;
+using Cesium.CodeGen.Extensions;
+using Cesium.CodeGen.Ir.Expressions;
+using Cesium.CodeGen.Ir.Expressions.Constants;
+using Mono.Cecil.Cil;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Cesium.CodeGen.Ir.BlockItems;
+
+internal class WhileStatement : IBlockItem
+{
+    private readonly IExpression _testExpression;
+    private readonly IBlockItem _body;
+    private readonly string? _breakLabel;
+
+    public WhileStatement(Ast.WhileStatement statement)
+    {
+        var (testExpression, body) = statement;
+
+        // 6.8.5.3.2 if testExpression is null it should be replaced by nonzero constant
+        _testExpression = testExpression?.ToIntermediate() ?? new ConstantExpression(new IntegerConstant("1"));
```

Comment:
Do we need this conversion in a `while` loop? It is only specified for the `for` loop. `while ()` (without the test expression) doesn't make any sense.